proto/src/build.rs

```diff
@@ -0,0 +1,7 @@
+use std::io::Result;
+
+fn main() -> Result<()> {
+    std::env::set_var("PROTOC", protobuf_src::protoc());
+    prost_build::compile_protos(&["proto/test_context.proto"], &["proto/"])?;
+    Ok(())
+}
```

Comment:
wrap calling protoc so that you don't have to do it. Similarly, if we used a build system then we would have it handle this piece.